src/components/FormikAutocomplete.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { InputLabel, TextField } from '@material-ui/core';
+import { Autocomplete } from '@material-ui/lab';
+import PropTypes from 'prop-types';
+
+/* eslint-disable react/jsx-props-no-spreading */
+function FormikAutocomplete(props) {
+  const { label, id, labelStyle, options, freeSolo, formik, placeholder } =
+    props;
+
+  return (
+    <>
+      <InputLabel className={labelStyle}>{label}</InputLabel>
+      <Autocomplete
+        id={id}
+        freeSolo={freeSolo}
+        multiple
+        value={formik.values.requirements[id]}
+        onChange={(e, newVal) =>
+          formik.setFieldValue(
+            `requirements.${id}`,
+            newVal,
+            /* shouldValidate = */ false
+          )
+        }
+        options={options}
+        renderInput={(params) => (
+          <TextField
+            {...params}
+            variant="outlined"
+            fullWidth
+            placeholder={
+              formik.values.requirements[id].length == 0 ? placeholder : ''
```

Comment:
<img width="405" alt="Screen Shot 2021-10-02 at 10 50 35 PM" src="https://user-images.githubusercontent.com/40483569/135741801-0cd66442-cf1d-43d7-b678-e022874f41f3.png">

unfortunately, it will keep the placeholder even if the TextField is not empty 🙃

Reply:
Ah darn, that's unfortunate. Let's keep it as is then. Maybe they fixed in MUIv5 and we can revisit this later.